run.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+failed=0
+
+build_all() {
+  sh -c 'rush build \
+    -t waffle-tutorial-hello-world \
+    -t waffle-tutorial-echo \
+    -t waffle-tutorial-token \
+    -t waffle-tutorial-nft'
+}
+
+rebuild_all() {
+  sh -c 'rush rebuild \
+    -t waffle-tutorial-hello-world \
+    -t waffle-tutorial-echo \
+    -t waffle-tutorial-token \
+    -t waffle-tutorial-nft'
+}
+
+test_all() {
+  examples=(
+    "hello-world"
+    "echo"
+    "token"
+    "nft"
+  )
+
+  ROOT=$(pwd)
+
+  for e in "${examples[@]}"
+  do
+    echo "--------------- testing waffle ${e} ---------------"
```

Comment:
Should we change the output to be more verbose (not significantly 😂 )?

```suggestion
    echo "--------------- Running Waffle tests ${e} ---------------"
```